doc/contributing/releases.md

```diff
@@ -102,7 +103,7 @@ gpg --armor --export email@server.com > ~/nodekey.asc
 Keys should be fetchable via:
 
 ```bash
-gpg --keyserver pool.sks-keyservers.net --recv-keys <FINGERPRINT>
+gpg --keyserver hkps://keys.openpgp.org --recv-keys <FINGERPRINT>
```

Comment:
Why is this change made here? I couldn't find any public reasoning for this.

Reply:
@anonrig FYI https://github.com/nodejs/node/issues/29531